app/helpers/percentage.js

```diff
@@ -0,0 +1 @@
+export { default, percentage } from 'ember-upf-utils/helpers/percentage';
```

Comment:
Are you sure of that ?

Reply:
didn't see that one 😅

Reply:
@ramshorst this should be removed IMO

Reply:
@tgallice If I remove this file it does not work any more. What does work is remplacing this file with the one from /addons. But that is done for none of the other helpers so that seams weird.

So, what do you mean ?

Reply:
My bad, I didn't see that is a helper you have added :) 
lgtm